platform/darwin/src/NSString+MGLAdditions.h

```diff
@@ -26,6 +26,9 @@ NS_ASSUME_NONNULL_BEGIN
 
  Only supports scripts for languages used by Mapbox Streets.
 
+ On iOS 8 or older, this will method will always return the untransliterated
```

Comment:
Same as above, it may be better use `@note` as a way to highlight the incompatibilities.